podcasts/Settings.swift

```diff
@@ -784,6 +784,21 @@ class Settings: NSObject {
         }
     }
 
+    // MARK: - Autoplay
+
+    static var autoplay: Bool {
+        set {
+            UserDefaults.standard.set(newValue, forKey: Constants.UserDefaults.autoplay)
+        }
+        get {
+            guard let isEnabled = UserDefaults.standard.object(forKey: Constants.UserDefaults.autoplay) as? Bool else {
+                return true
```

Comment:
This means its going to be enabled by default, right? All good if so, but not sure I saw that in the project thread

Reply:
Yes, it is! I'll update this info there